packages/core/README.md

```diff
@@ -351,6 +351,33 @@ Additionally to the regular fields, `primaryContrast` and `secondaryContrast` ar
 }
 ```
 
+#### dark theme
+
+To enable an additional dark theme, simply add `dark` under the `light` theme as follows below:
+
+```js
+ theme: {
+    themes: {
+      light: {
+        primary: 'black',
+        primaryContrast: 'white',
+        secondary: '#c0ffee',
+        secondaryContrast: '#de1e7e',
+      },
+      dark: {
+        primary: 'white',
+        primaryContrast: 'black',
+        secondary: '#de1e7e',
+        secondaryContrast: '#c0ffee',
+      },
+    },
+  },
+```
+
+The theme can either be activated directly by setting `dark: true` within the `theme`-object or by calling the `setTheme` method within the store.
```

Comment:
```suggestion
```
I don't think that this is necessary. If someone wants to have their client in dark mode from the getgo, they'll just look it up in the vuetify configuration. If they want to change it later, they can look at the mutation description